platform/app/src/components/ViewportGrid.tsx

```diff
@@ -208,8 +213,24 @@ function ViewerViewportGrid(props: withAppTypes) {
   }, [viewports]);
 
   const onDropHandler = (viewportId, { displaySetInstanceUID }) => {
-    const updatedViewports = _getUpdatedViewports(viewportId, displaySetInstanceUID);
-    viewportGridService.setDisplaySetsForViewports(updatedViewports);
+    let dropHandlerPromise = Promise.resolve({ handled: false });
+    const customOnDropHandler = customizationService.getCustomization('customOnDropHandler');
```

Comment:
you need to provide a default null in the customizations folder in the extension. Also document it in here https://docs.ohif.org/platform/services/customization-service/customizationService#customizable-parts-of-ohif